examples/generative/corrdiff/training/training_loop.py

```diff
@@ -224,13 +225,16 @@ def training_loop(
                 img_lr = img_lr.to(device).to(torch.float32).contiguous()
                 labels = labels.to(device).contiguous()
 
-                loss = loss_fn(
-                    net=ddp,
-                    img_clean=img_clean,
-                    img_lr=img_lr,
-                    labels=labels,
-                    augment_pipe=augment_pipe,
-                )
+                with torch.autocast(
+                    "cuda", dtype=torch.bfloat16, enabled=(fp_mode == "amp")
```

Comment:
Why always enable `bfloat16`? This should be configurable ideally.

Reply:
Implemented in https://github.com/NVIDIA/modulus/pull/423/commits/de4f91833da024d35098009f09cca8a1808a2784.